### DISCUSSION TITLE
Octokit.net SearchIssuesRequest - Is there a way to search for issues that have not been locked?

### DISCUSSION THREAD

**Discussion by JimSuplizio** [octokit/octokit.net#2615] [Q&A] [answered]
I'm using Octokit.net's SearchIssuesRequest for several things, and it's pretty awesome, one thing I'm having trouble with is figuring out how to search for issues that are locked/unlocked. One of the things I need to do for closed issues older than X days is lock them. We have several thousand closed issues but I haven't been able to figure out how to limit the search for only those issues that haven't been locked and I was wondering if this can be done with SearchIssuesRequest?

**Comment by nickfloyd** [comment, marked as answer]
Hey @JimSuplizio, Thanks for reaching out! ❤️ 

So I've got some good and bad news for you...
- The good news is that filtering issues based on if they are locked or not is possible through the API - [doc](https://docs.github.com/en/search-github/searching-on-github/searching-issues-and-pull-requests#search-based-on-whether-a-conversation-is-locked)
- The bad news is that it has not yet been implemented in the .NET SDK - [source](https://github.com/octokit/octokit.net/blob/main/Octokit/Models/Request/SearchIssuesRequest.cs)

Please feel free to open a PR to implement this, and let us know how we can help! We'll create an [issue](https://github.com/octokit/octokit.net/issues/2620) for this request/question for prioritization and tracking purposes - it would be a great addition to the Issue Search API.

**Reply by JimSuplizio** [reply]
Thank you @nickfloyd for confirming this and creating the work item.
